src/roles/common/tasks/check-predeploy-prereq.yml

```diff
@@ -71,6 +71,24 @@
 
   when: not vsd_inplace_upgrade | default(false)
 
+- name: Check for portal files missing
+  set_fact:
+    is_portal_path_missing: "{{ portal_path | default('MISSING') == 'MISSING' }}"
+    is_portal_file_missing: "{{ portal_file_name | default('MISSING') == 'MISSING' }}"
+
+- name: Delete the md5 files
+  file:
+    path: "{{ inventory_dir }}/deployment.md5.yml"
+    state: absent
+  when: is_portal_file_missing or is_portal_path_missing
+  delegate_to: localhost
+
+- name: Check the image path for portal container
+  assert:
+    that: not is_portal_file_missing or not is_portal_path_missing
+    msg: "Missing SDWAN portal container tar or CentOS image for portal under the unzipped directory, Please Copy it"
+  when: component_type | default("NONE") == 'portal'
```

Comment:
I think this component_type check needs to be extended to the other tasks that were added using a block.

Reply:
done